test/test_nn.py

```diff
@@ -1565,12 +1566,13 @@ def _test_EmbeddingBag(self, cuda, mode, sparse, dtype=torch.double):
                  [3, 4]], device=device, dtype=dtype)
 
         output = es(input, offsets)
-        output.backward(grad_output)
+        output.backward(grad_output if cuda else grad_output_with_empty)
```

Comment:
this looks wrong (maybe wrong merge conflict?). There's no `grad_output_with_empty` in this file.

Reply:
This also slipped through from D7875031. See the comment thread on aten/src/ATen/native/EmbeddingBag.cpp line 28